cmd/opm/main.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import (
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+
+	cli "github.com/operator-framework/operator-registry/cmd/opm/registry"
+)
+
+func main() {
+	rootCmd := &cobra.Command{
+		Use:   "opm",
+		Short: "operator package manager",
+		Long:  "Top level CLI for operator-registry",
```

Comment:
Is this long description accurate anymore?

Reply:
addressed